corehq/apps/users/forms.py

```diff
@@ -379,6 +379,110 @@ def clean(self):
     validate_username = EmailValidator(message=ugettext_lazy(u'Username contains invalid characters.'))
 
 
+_username_help = """
+<span ng-if="usernameAvailabilityStatus === 'pending'">
+    <i class="fa fa-circle-o-notch fa-spin"></i>
+    %(checking)s
+</span>
+<span ng-if="usernameAvailabilityStatus === 'taken'"
+      style="word-wrap:break-word;">
+    <i class="fa fa-remove"></i>
+    {{ usernameStatusMessage }}
+</span>
+<span ng-if="usernameAvailabilityStatus === 'available'"
+      style="word-wrap:break-word;">
+    <i class="fa fa-check"></i>
+    {{ usernameStatusMessage }}
+</span>
+<span ng-if="usernameAvailabilityStatus === 'error'">
+    <i class="fa fa-exclamation-triangle"></i>
+    %(server_error)s
+</span>
+""" % {
+    'checking': ugettext_noop('Checking Availability...'),
+    'server_error': ugettext_noop('Issue connecting to server. Check Internet connection.')
+}
```

Comment:
This had me pretty confused for a minute there, with the mixed string formatting markup - what do you think about using something other than `{{` and `}}` for angular templating? https://docs.angularjs.org/api/ng/provider/$interpolateProvider
(not blocking)